kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/util/PipelineDistributedBarrier.java

```diff
@@ -96,9 +102,10 @@ public void persistEphemeralChildrenNode(final String parentPath, final int shar
     /**
      * Persist ephemeral children node.
      *
-     * @param parentPath parent path
+     * @param jobId job id
      */
-    public void removeParentNode(final String parentPath) {
+    public void unregister(final String jobId) {
+        String parentPath = PipelineMetaDataNode.getJobBarrierDisablePath(jobId);
```

Comment:
`register` method has hard-coded `enable` path, and this `unregister` method use `disable` path, is it expected?